packages/material-ui/src/Tabs/Tabs.js

```diff
@@ -338,10 +338,12 @@ const Tabs = React.forwardRef(function Tabs(props, ref) {
     };
   }, [updateIndicatorState, updateScrollButtonState]);
 
-  const handleTabsScroll = React.useCallback(
-    debounce(() => {
-      updateScrollButtonState();
-    }),
+  const handleTabsScroll = React.useMemo(
+    () =>
+      debounce(() => {
+        updateScrollButtonState();
+      }),
```

Comment:
Would you be opposed to me following up with such a diff to avoid creating another function (marginal bundle size win and no creation of an intermediary function)?

```suggestion
      debounce(updateScrollButtonState),
```

And we can probably just use an `[]` (with an `eslint-disable` comment) instead of `[updateScrollButtonState]` because it would never change due to `useEventCallback`

Reply:
> And we can probably just use an [] (with an eslint-disable comment) instead of [updateScrollButtonState] because it would never change due to useEventCallback

Works now, may not work in the future. Let's not be too clever to safe 4 bytes.

Reply:
Sure. What about the extra function? Doesn't require any `eslint-disable` comments

Reply:
Not sure where an extra function is. Open a PR so that we're on the same page?